src/webgl/p5.Camera.js

```diff
@@ -127,9 +127,29 @@ p5.prototype.camera = function (...args) {
  * vertical field of view, aspect ratio (usually width/height), and near and far
  * clipping planes.
  *
- * If no parameters are given, the following default is used:
- * perspective(PI/3, width/height, eyeZ/10, eyeZ*10),
- * where eyeZ is equal to ((height/2) / tan(PI/6)).
+ * If no parameters are given, the default values are used as:
+ *
+ * fov- The default field of view for the camera is such that the full height of renderer is visible when it is positioned at a default distance of 800 units from the camera.
```

Comment:
I think we can maybe improve the readability of this list, perhaps by turning it into a bullet list?
<img width="1022" alt="image" src="https://github.com/processing/p5.js/assets/5315059/b9b21d6f-d379-4e90-ae9d-7ba6d2eaf772">


To do so, we need to put a `- ` before each item, and take out the empty lines between them. I also think it might flow better once in a bullet list if we use a colon afterwards instead of a dash. Lastly, since each of these is a variable name, we can put them in backticks:
<img width="197" alt="image" src="https://github.com/processing/p5.js/assets/5315059/0772510b-8b34-4e85-8f87-1eb3246e4f6f">